jenkins/vm-reset

```diff
@@ -61,18 +61,35 @@ validate_args() {
 
 wait_till() {
     local vm_id=$1
-    local state=$2
+    local action_type=$2
+    local interval
+    local nr_steps
+
+    if [ $action_type == "power" ]; then
```

Comment:
I don't see the need for `action_type`, 
- just pass the `interval` secs
- calculate `nr_steps` based on the `interval`.